code/cansat_code/wind/wind.py

```diff
@@ -0,0 +1,34 @@
+import Adafruit_ADS1x15
+import time
+adc = Adafruit_ADS1x15.ADS1115()
+GAIN = 1
+DATARATE = 128
+ENC_NUM = 3 #may be 2 or 3
+SAMPLE_SIZE = 5
+BORDER_VALUE = 20000
+
+def read_state():
+    value = adc.read_adc(ENC_NUM, gain=GAIN, data_rate=DATARATE)
+    if value<BORDER_VALUE:
+        return 0
+    else:
+        return 1
+
+
+def read_time():
+    primary_time=time.time()
+    primary=read_state()
+    while read_state() == primary:
```

Comment:
to się może nigdy nie skończyć jeżeli nie będzie się obracać? trzebaby jakiś timeout na to założyć, coś w stylu:
```
if time.time() - primary_time > 30:
    logging.warning('wind sensor is not working or working super slow')
```